[DSS-274] Customize storybook (#18)
Kajabi/sage: libs/core/src/stories/Welcome.stories.mdx

import { Meta, Story } from '@storybook/addon-docs';

<Meta title="Welcome" parameters={{ previewTabs: { canvas: { hidden: true }, } }} />

# Building with Sage
Our system is our source of truth, providing everything you need to build the best experiences for all Kajabi entrepreneurs.

## Foundations
Our system starts with some core visual elements: Type, Color, and Icons. We also prescribe common practices for aspects of the system.

## Components

> Components are the backbone of our design system. In order to be most usable, our components are written in both Rails and React form.

Reviewer: >Components are the backbone of our design system. In order to be most usable, our components are written in both Rails and React form.

This is not the case...we are building one code base using Web Components

Code examples, property options, and best practices can all be discovered here. Links to Figma design documentation can also be found in this section.

## Contributing

> We encourage any member of the Kajabi team to feel empowered to participate. Understanding and contributing to the system are the keys to the system's longevity.

Reviewer: Are we only building this for Kajabi? If so, lets make the repo private

cc: @monicawheeler

Reviewer 2: So far! :nitpick 🤣

Reviewer 3: We want our Core Design System repo (this repo) to be public facing, so our language in documentation should be geared that way.

```suggestion
We encourage everyone to feel empowered to participate. Understanding and contributing to the system are the keys to the system's longevity.
```
